iterations_runners/iterations_runner.rb

```diff
@@ -31,38 +28,57 @@ def assert(cond)
     param = Integer(param)
     debug = Integer(debug) > 0
 
-    assert benchmark.end_with?(".rb")
+    if ! benchmark.end_with?(".rb")
```

Comment:
Is there any particular reason to a) make this something we check b) make it more complex than an assertion?